crates/codegen/runtime/cargo/crate/src/runtime/compilation/internal_builder.rs

```diff
@@ -43,7 +44,10 @@ impl InternalCompilationBuilder {
 
         let import_paths = self.imports.extract(parse_output.create_tree_cursor());
 
-        let file = File::new(id.clone(), parse_output.tree().clone());
+        let file = File::new(
```

Comment:
Should `File` have a `Nonterminal` instead of a `Node`?

Reply:
Good catch. That sounds great.

Reply:
I added it in https://github.com/NomicFoundation/slang/pull/1187/commits/850b5f90bed24bbd1a97c22dfa2912d1186d27d7